bucket/registryfinder.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": "2.52.0.0",
+    "description": "Registry Finder is an improved replacement for the built-in Windows registry editor",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://registry-finder.com/License.txt"
+    },
+    "homepage": "https://registry-finder.com/",
+    "notes": "Run \"Registry Finder\" from the Start Menu, or \"registryfinder.exe\" from the command line.",
+    "architecture": {
+        "64bit": {
+            "url": "https://registry-finder.com/bin/2.52.0.0/RegistryFinder64.zip",
+            "hash": "f57b29e1f6f6f03af05332ce6897c1234a1cf580c8b481f27a689765acc4251d"
+        },
+        "32bit": {
+            "url": "https://registry-finder.com/bin/2.52.0.0/RegistryFinder.zip",
+            "hash": "d8255947a1d7e3be07cdc9021c945afc2d67d7ec5149c5262d05ad044878695a"
+        }
+    },
+    "bin": [
+        [
+            "registryfinder.exe",
+            "registryfinder",
+            "--dataFolder $persist_dir"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "RegistryFinder.exe",
+            "Registry Finder",
+            "--dataFolder $persist_dir"
```

Comment:
Same as above

Reply:
It works.  Is it not supposed to?

Reply:
If there's nothing to persist (no `persist` field), $persist_dir will not exist

Reply:
There is nothing to persist until the program is executed the first time. The software must create the folder then.  Is it possible to create an empty `$persist_dir`? Probably can be done using a `pre_install` script.

Reply:
In this case RegistryFinder itself is creating the folder. 

----

> ![image](https://user-images.githubusercontent.com/46838874/148699098-5e3cb19b-9e9d-45b8-ab27-5964f67ed76b.png)
> 
> I think just persisting RegistryFinder.config should be enough (instead of entire folder)

What about this?

Reply:
THat would work I guess.  There are however other files that are created in the persist folder:

```
│   favorites.txt
│   RegistryFinder.config
│
└───UndoHistory
        UndoHistory.dat
        UndoHistory.desc.dat
```

Reply:
I prefer a native persist method, i.e., persist a folder called, e.g., `config`, and let bin and shortcut use that folder as dataFolder `--dataFolder \"$dir\\config\"`.

Tested on my PC.

Reply:
```json
    "bin": [
        [
            "registryfinder.exe",
            "registryfinder",
            "--dataFolder \"$dir\\config\""
        ]
    ],
    "shortcuts": [
        [
            "RegistryFinder.exe",
            "Registry Finder",
            "--dataFolder \"$dir\\config\""
        ]
    ],
    "persist": "config",
```

Reply:
Yes, this method sounds cleaner